### DISCUSSION TITLE
Nested List for values

### DISCUSSION THREAD

**Discussion by Ilijan91** [afconsult/quill-mention#298] [Ideas]
Is there a possibility of having nested values as dropdowns?
Hi is there a possibility to have a nested array of values instead of a single value array
Something like this ->
`[
  { id: 2, value: { label: 'Product', values: ['Product 1', 'Product 2'] } },
  { id: 2, value: { label: 'Customer', values: ['Customer 1', 'Customer 2'] } }
];`
and at the end choose Product 1, or Product 2 as a single value

something like this

<img width="380" alt="Screenshot 2022-08-31 at 12 07 57" src="https://user-images.githubusercontent.com/44878732/187654787-07b5d47d-7068-47c6-87d3-7e78a44ba019.png">

**Comment by alanltr-bob** [comment]
Hi @Ilijan91 , have you found a trick to make or simulate this kind of mention?
